### PR TITLE
Bump spark version to avoid vulnerabilities

### PR DESCRIPTION
**Pull request by shiyuhang0**
### What problem does this PR solve? 
This pr is a supplement to https://github.com/pingcap/tispark/pull/2486. It bumps the Spark version：
- from 3.0.2 to 3.0.3
- from 3.1.1 to 3.1.3
- from 3.2.1 to 3.2.2

Note that spark 3.0.3 still has vulnerabilities

### CODE REVIEW

**Review by zhangyangyu — APPROVED** [review]

**Review by xuanyu66 — CHANGES_REQUESTED** [review]
https://github.com/pingcap/tispark/blob/master/pom.xml#L183-L209
Bump these too

**Comment by xuanyu66** [comment]
And the better solution is to define variables so that we won't miss the places need to be modified

**Review by Daemonxiao — APPROVED** [review]

**Comment by shiyuhang0** [comment]
> https://github.com/pingcap/tispark/blob/master/pom.xml#L183-L209 Bump these too

has done in https://github.com/pingcap/tispark/pull/2486

File: pom.xml [hunk 1]
```diff
@@ -73,6 +73,10 @@
         <spark.version.compile>3.0.3</spark.version.compile>
         <spark.version.test>3.0.3</spark.version.test>
         <spark.version.release>3.0</spark.version.release>
```
Comment (zhangyangyu) [inline pom.xml:75]: still need or not?

Comment (shiyuhang0) [reply pom.xml:75]: we need it for it is the default value

Comment (shiyuhang0) [reply pom.xml]: I have optimized it

**Review by xuanyu66 — APPROVED** [review]

**Pull request merged**